src/Vlingo.Wire.Tests/Fdx/Inbound/MockInboundStreamInterest.cs

```diff
@@ -30,15 +30,14 @@ public void HandleInboundStreamMessage(AddressType addressType, RawMessage messa
             var textMessage = message.AsTextMessage();
             TestResult.Messages.Add(textMessage);
             TestResult.MessageCount.IncrementAndGet();
-            Debug.WriteLine($"INTEREST: {textMessage} list-size: {TestResult.Messages.Count} count: {TestResult.MessageCount.Get()} count-down: {TestResult.UntilStops.Remaining}");
```

Comment:
You don't need to get rid of that one. You can pass into the constructor the expected number and calculate it substracting `TestResult.UntilStops.TotalWrites`